test_conformance/spirv_new/test_op_vector_extract.cpp

```diff
@@ -76,27 +87,30 @@ int test_extract(cl_device_id deviceID, cl_context context,
     return 0;
 }
 
-#define TEST_VECTOR_EXTRACT(TYPE, N)                        \
-    TEST_SPIRV_FUNC(op_vector_##TYPE##N##_extract)          \
-    {                                                       \
-        typedef cl_##TYPE##N Tv;                            \
-        typedef cl_##TYPE Ts;                               \
-        const int num = 1 << 20;                            \
-        std::vector<Tv> in(num);                            \
-        const char *name = "vector_" #TYPE #N "_extract";   \
-                                                            \
-        RandomSeed seed(gRandomSeed);                       \
-                                                            \
-        for (int i = 0; i < num; i++) {                     \
-            in[i] = genrand<Tv>(seed);                      \
-        }                                                   \
-                                                            \
-        return test_extract<Tv, Ts>(deviceID,               \
-                                    context, queue,         \
-                                    name,                   \
-                                    in, N);                 \
+#define TEST_VECTOR_EXTRACT(TYPE, N)                                           \
+    TEST_SPIRV_FUNC(op_vector_##TYPE##N##_extract)                             \
+    {                                                                          \
+        if (sizeof(cl_##TYPE) == 2)                                            \
```

Comment:
That'll be a nice surprise if anyone ever adds uint16 to this test, but I don't see an easy alternative.